src/main/java/org/sert2521/powerup/drivetrain/Drivetrain.kt

```diff
@@ -40,7 +41,10 @@ object Drivetrain : Subsystem() {
         rightDrive.setSelectedSensor(FeedbackDevice.QuadEncoder)
     }
 
-    override fun onStart() = reset()
+    override fun onStart() {
+        reset()
+        TurnToAngle(180.0).start()
```

Comment:
🤔

Reply:
Yeah, I was planning on rebasing it out. The PRs are mostly just there so I remember cuz I'll probably merge them locally anyway.